tests/slr_jest/blockEditor/hook-callbacks.spec.js

```diff
@@ -791,4 +792,44 @@ describe('hook-callbacks', () => {
 			);
 		});
 	});
+
+	describe('filterSettingsFields', () => {
+		it('should return layout select component if service is ok', () => {
+			select.mockReturnValue({
+				isServiceStatusOk: () => true,
+				getCurrentLayoutId: () => 'layout-uuid-1',
+				getLayoutsInOptionFormat: () => [
+					{
+						value: 'layout-uuid-1',
+						label: 'Layout Name',
+					},
+				],
+			});
+
+			const fields = filterSettingsFields([]);
+
+			// fields should be an array with one item.
```

Comment:
Are these comments required? The code that follows them is pretty self-explanatory, I think.
If the comments are required, then they should `// Start with an upper-case letter and end with a full-stop.`